messages/ctrlMsg.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package mateROV;
+
```

Comment:
You also need to add this message to the MsgType enum in the `__init__.py` file in the messages directory.